net/interfaces/interfaces.go

```diff
@@ -503,9 +531,8 @@ func isPrivateIP(ip netaddr.IP) bool {
 	return private1.Contains(ip) || private2.Contains(ip) || private3.Contains(ip)
 }
 
-func isGlobalV6(ip netaddr.IP) bool {
-	return v6Global1.Contains(ip) ||
-		(tsaddr.IsULA(ip) && !tsaddr.TailscaleULARange().Contains(ip))
+func isUsableV6(ip netaddr.IP) bool {
+	return v6Global1.Contains(ip) || tsaddr.IsULA(ip)
```

Comment:
Add a comment to this function? Why is a Tailscale ULA address "usable"? In what way? It must be for something if this PR removes the `&& !tsaddr.TailscaleULARange().Contains(ip)` part from line 508.

Reply:
The PR has evolved back to the point where there isn't a need to change it to omit TailscaleULARange, so I put it back. 

Arguably the name should have been changed back when ULA addresses were first allowed a couple months ago, so now we're just changing the name to suit.